### ISSUE TITLE
Process halting on Windows 10 When Out Of Focus

### ISSUE THREAD

**Issue by glenndilen** [hummingbot/hummingbot#2995]
**Describe the bug**
Whenever using the windows version on windows 10 - once Hummingbot is no longer the focus window, the process halts.
When the window is brought back into focus, and spacebar (or any other key is pressed) all actions that should have happened when the window was out of focus - happens all at once.

This results in multiple buy/sell orders being placed and cancelled at once - as well as all of the telegram notifications coming through at once.

**Steps To Reproduce**
Simple start Hummingbot on Windows 10, use some other software and do not focus on the Hummingbot window.
You will notice the bot will stop doing anything after about 10-15 minutes - until the user focuses on the window, and presses a key input.

Here are the steps to reproduce the issue (see attachments in the section below):

1. Set up your configuration file with these parameters: Any parameters. A quick order refresh rate would help to display the problem.
1. Start the bot using this command: start

**Release version**
0.36.0

**Optional: your discord username:**
null

┆Issue is synchronized with this [Clickup task](https://app.clickup.com/t/1ram7wj) by [Unito](https://www.unito.io)

**Comment by rapcmia** [comment]
I tried to replicate this by running a Binary build (v0.36.0) on Win10 and created a simple liquidity_mining with order refresh 10s. Minimized the bot and work on another task. Checked the bot again after 15mins bot is still running ok, Telegram will not spam me for updates unless use `config` on the bot then it will send me messages on it. 

I assume you have tried to reinstall Hummingbot and see if this happens again? We have other Hummingbot installation options for Win10, you may want to try docker found on this link: https://docs.hummingbot.io/installation/windows/#install-via-docker

**Comment by PtrckM** [comment]
Can you tell which type of Windows 10 Edition are you using? We cannot replicate this using Windows 10 Home and Windows 10 Enterprise.

**Comment by glenntomas** [comment]
I tried replicating a user issue related to this and I think I replicated it. We have a [Zendesk Ticket](https://hummingbot.zendesk.com/agent/tickets/534) related to this ticket.

**Steps To Reproduce**
1. Create a bot using Binary Build
2. Choose Pure Market Making
3. Start the bot
4. Do other things and just run the bot in the background

- I run `status --live` before doing other things and when I came back to check the bot it is stuck `1:13:38`

![4dMCyxDbmP](https://user-images.githubusercontent.com/73885708/127969304-e7275ffb-1d4d-4e60-b3f7-5c96c5da8e5c.gif)

- When I click or type on the bot, it starts to run again

![M4J1KCwMMR](https://user-images.githubusercontent.com/73885708/127969933-52ef4c56-41cf-4dd7-a099-102a9d2c2a99.gif)

**PC Specifications**
Processor: AMD Ryzen 5 2600 Six-Core Processor 3.40 GHz
Installed RAM:	16.0 GB
System type: 64-bit operating system, x64-based processor
Edition: Windows 10 Pro
Version: 20H2
OS build: 19042.1110
Experience: Windows Feature Experience Pack 120.2212.3530.0

**Release Version**
v0.41.0
